packages/domain-picker/src/domain-picker/index.tsx

```diff
@@ -182,7 +159,7 @@ const DomainPicker: FunctionComponent< Props > = ( {
 				isLink
 				className="domain-picker__cancel-button"
 				onClick={ () => {
-					onCancel && onCancel();
+					onClose();
 				} }
```

Comment:
NIT: This is almost synonymous to the following. I think we can a wrapping function:

```js
onClick={ onClose }
```

Reply:
Thanks! Done.